test/plugins/features/movies/index.test.js

```diff
@@ -18,14 +28,14 @@ describe('movies integration', () => {
       });
     });
 
-    it('gets movies prior to a release_year', () => {
+    it('gets movies after year_start (inclusive)', () => {
```

Comment:
Since this is testing the list endpoint, you should create a new `describe` block for `list`.